src/test/java/tatracker/testutil/TaTrackerBuilder.java

```diff
@@ -28,6 +29,14 @@ public TaTrackerBuilder withStudent(Student student) {
         return this;
     }
 
+    /**
+     * Adds a new module to the ta tracker.
+     */
+    public TaTrackerBuilder withModule(Module module) {
+        taTracker.addModule(module);
+        return this;
+    }
+
```

Comment:
No with Group?

Reply:
Cause groups are inside modules, not inside the ta-tracker so the ta tracker shouldn't keep track of groups.